.circleci/docker/ubuntu/Dockerfile

```diff
@@ -122,12 +132,19 @@ ADD ./common/install_jni.sh install_jni.sh
 ADD ./java/jni.h jni.h
 RUN bash ./install_jni.sh && rm install_jni.sh
 
+# Install Open MPI for CUDA
+ADD ./common/install_openmpi.sh install_openmpi.sh
+RUN if [ -n "${CUDA_VERSION}" ]; then bash install_openmpi.sh; fi
+RUN rm install_openmpi.sh
+
 # Include BUILD_ENVIRONMENT environment variable in image
 ARG BUILD_ENVIRONMENT
 ENV BUILD_ENVIRONMENT ${BUILD_ENVIRONMENT}
 
-# Install LLVM dev version (Defined in the pytorch/builder github repository)
-COPY --from=pytorch/llvm:9.0.1 /opt/llvm /opt/llvm
```

Comment:
This seems to be the root-cause of the bug. The build of the PyTorch wheel used in the failing CI check is unable to find LLVM, which is used by TensorExpr.
So, the lines corresponding to those unexpectedly passing tests that I alluded to earlier need not be removed, then.